package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "rdpc-ui",
-  "version": "0.4.0",
+  "version": "0.5.0",
   "private": true,
   "scripts": {
     "dev": "next dev",
-    "build": "next build",
+    "build": "NEXT_IS_BUILDING=true next build",
```

Comment:
ooh new flag. they just love polluting the env, eh?

Reply:
that's me polluting the env ;)

Reply:
and yes I do enjoy it :p